docs/README-dev.md

```diff
@@ -58,11 +58,15 @@ Because of the above, include paths in source should always feature the full pat
 Note that the recommended way of building the project is to use separate build directory, where CMake will generate all targets. In this way, you can keep a clean source directory. The following shell commands illustrate this usage:
 
 ```sh
-  cd <colobot_repository>
+  git clone https://github.com/colobot/colobot.git
+  cd colobot
+  git submodule update --init
   mkdir build
-  cmake ../
+  cd build
+  # You can't compile translations unless you have python2
+  cmake -D TRANSLATIONS=OFF -D COLOBOT_DEVELOPMENT_MODE=ON ..
```

Comment:
Progress on removing -D TRANSLATIONS=OFF: blocked waiting for https://github.com/colobot/colobot-data/pull/116 to be merged

Reply:
Do we need https://github.com/colobot/colobot-data/pull/116 merged for this pull request? It fixes translations but it seems to be an unrelated fix. We need to update data submodule instead.

Reply:
Status:
1. I think #1616 **can** be merged as is **with** -D TRANSLATIONS=OFF
2. You are right: before  we remove -D TRANSLATIONS=OFF we need to update the data submodule
3. But before we update the data submodule we need to merge https://github.com/colobot/colobot-data/pull/116 because currently running `make` with the most recent version of the data submodule and translations enabled results in some files in data/ being automatically modified